src/config/enterprise.rs

```diff
@@ -55,53 +55,71 @@ static DATADOG_REPORTING_PATH_STUB: &str = "/api/unstable/observability_pipeline
 pub static DATADOG_API_KEY_ENV_VAR_SHORT: &str = "DD_API_KEY";
 pub static DATADOG_API_KEY_ENV_VAR_FULL: &str = "DATADOG_API_KEY";
 
-#[derive(Debug, PartialEq, Clone)]
+/// Enterprise options for using Datadog's Observability Pipelines.
```

Comment:
Thinking longer-term, would be helpful to provide a link to OP here?

Reply:
Hmm, good point.

Reply:
Fixed in 8b55d4493.